android/app/src/main/AndroidManifest.xml

```diff
@@ -78,6 +79,22 @@
             <action android:name="android.appwidget.action.APPWIDGET_CONFIGURE" />
           </intent-filter>
         </activity>
+
+        <receiver android:name=".NewSmallWidgetProvider" android:label="@string/new_small_widget" android:exported="false">
```

Comment:
Yep, I just wanted to keep the old files for reference for a short while still.

Reply:
I think that it is not necessary, because the git keeps the old code. All new code should be good quality and I think that using new-prefixes is not good practice, because something is new just short time (then it will become current or old). Better naming would be to use version numbers `small_widget_v2` if necessary.

Reply:
But maybe it is time to delete the old files and remove the new-prefxes.
I do that now...

Reply:
Done